lib/reek/cli/utilities.rb

```diff
@@ -0,0 +1,12 @@
+module Reek
+  module CLI
+    # CLI utilities
+    module Utilities
+      def self.require_silently
+        old_verbose, $VERBOSE = $VERBOSE, nil
+        yield if block_given?
```

Comment:
I would just require a block and drop the `block_given?` part: If there's no block this method is pretty useless :laughing:.